django_rq/templates/django_rq/clear_queue.html

```diff
@@ -9,7 +11,7 @@
             width: 80%;
         }
     </style>
-    <link href="{{ STATIC_URL }}admin/css/forms.css" type="text/css" rel="stylesheet">
+    <link href="{% static 'admin/css/forms.css' %}" type="text/css" rel="stylesheet">
```

Comment:
It didn't work for me, and since you've been using the `{% static %}` tag everywhere else I thought it might be a good idea to make it consistent.